flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/BaseReconciler.java

```diff
@@ -41,17 +43,32 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(BaseReconciler.class);
 
-    public static final int REFRESH_SECONDS = 60;
-    public static final int PORT_READY_DELAY_SECONDS = 10;
+    public final int reconcileDelayInSec;
+    public final int portCheckIntervalInSec;
```

Comment:
Maybe these could be captured into it's own configclass for easy access and that way it's going to be much easier to extend in the future